index.js

```diff
@@ -673,9 +673,15 @@ export default e => {
     using = e.use;
   });
 
-  useFrame(() => {
-    const localPlayer = useLocalPlayer();
+  const localPlayer = useLocalPlayer();
+  const meleewhoosh = (e) =>{
+    if(e.data.indexOfCombo === localPlayer.avatar.useAnimationIndex){
+      sounds.playSound(soundFiles.combat[soundIndex + (4 * e.data.indexOfCombo + Math.floor(Math.random() * 4))]);
```

Comment:
I don't get this math. How is it supposed to work? Isn't this going to be out of bounds?

Reply:
Thanks for pointing out, will check

Reply:
Each combo has 4 sounds

Reply:
Ok I see. This is definitely not a constant that belongs here then.

The animation selection should not be hidden in an external module (`silsword`). And this constant will change so it needs to be an actual constant.

This is really not good code...

Reply:
All of this selection logic must live in the place the animations live, in `app`.

Reply:
Got it. That make sense

https://github.com/webaverse/app/pull/3660/commits/db697da9e8f7d743cccbf54e3ec148ac21013013

https://github.com/webaverse/silsword/pull/12/commits/82451c636b6c1114570d2fa98b5e3c1f28570a7a